docs/content/guides/dapp-deployment/testnet-deployment.md

```diff
@@ -0,0 +1,164 @@
+
+You'll receive an email once your request is approved and the tokens are deposited.
+
+Likewise, you'll receive an email if there were any problems with your request.
+
+### Accessing Flow Testnet
+
+Use the following host address to programmatically connect to Flow Testnet:
+
+- `access.devnet.nodes.onflow.org:9000`
+
+Go SDK Example:
+
+```go
+import "github.com/onflow/flow-go-sdk/client"
+
+func main() {
+  flowAccessHost := "access.devnet.nodes.onflow.org:9000"
+  flowClient, _ := client.New(flowAccessHost, grpc.WithInsecure())
+
+  // ...
+}
+```
+
+### Deploying a Contract
+
+Using the account you created above, you can deploy additional contract accounts using the [Flow CLI](https://github.com/onflow/flow-cli).
+
+Start by initializing you Flow project if you haven't already:
+
+```sh
+flow init
+```
+
+This will create a `flow.json` file in the current directory. It should look like this:
+
+```json
+{
+  "accounts": {
+    "service": {
+      "address": "f8d6e0586b0a20c7",
+      "privateKey": "<EMULATOR PRIVATE KEY>",
+      "sigAlgorithm": "ECDSA_P256",
+      "hashAlgorithm": "SHA3_256"
+    }
+  }
+}
+```
+
+The `service` account entry is auto-generated for use with the [Flow Emulator](/emulator).
+
+You'll want to add an account entry for your new testnet account:
+
+```json
+{
+  "accounts": {
+    "service": {
+      "address": "f8d6e0586b0a20c7",
+      "privateKey": "<EMULATOR PRIVATE KEY>",
+      "sigAlgorithm": "ECDSA_P256",
+      "hashAlgorithm": "SHA3_256"
+    },
+    "testnet": {
+      "address": "<YOUR TESTNET ADDRESS>",
+      "privateKey": "<YOUR TESTNET PRIVATE KEY>",
+      "sigAlgorithm": "ECDSA_P256",
+      "hashAlgorithm": "SHA3_256"
+    }
+  }
+}
+```
+
+After this, use the `accounts create` command to deploy a new contract account:
+
+```sh
+flow accounts create \
+  --host access.devnet.nodes.onflow.org:9000 \
+  --signer testnet \
+  --code MyContract.cdc
+```
+
+- `host` is the access node to connect to
+- `signer` is the account entry used for signing
+- `code` is the path to the Cadence contract being deployed
+
+### Making Use of Core Contracts
+
+Flow Testnet comes with some useful contracts already deployed, called **core contracts.** There are reference pages with import addresses for the core contracts here: [https://docs.onflow.org/protocol/core-contracts/](/protocol/core-contracts/)
+
+Once you accounts are set up and you're ready to develop, you can look over some code examples from the Flow Go SDK here:
+
+- [https://github.com/onflow/flow-go-sdk/tree/master/examples](https://github.com/onflow/flow-go-sdk/tree/master/examples)
+
+### Redeploying Your Application
+
+If you discover your application is broken after an update, use the latest emulator to test changes to your application. Once you are satisfied that you've patched any breaking changes, you'll need to get in touch with the Flow core team to redeploy your contracts to Testnet.
```

Comment:
So I know we don't need approval for deployment, but are we still requiring Service Account for re-deployment? A quick look through flow-go indicates that it should be off for testnet but was 100%. cc @psiemens